src/main/java/core/basesyntax/service/RegistrationService.java

```diff
@@ -3,5 +3,5 @@
 import core.basesyntax.model.User;
 
 public interface RegistrationService {
-    User register(User user);
+    User register(User user) throws RegistrationUserException;
```

Comment:
this is redundant

Reply:
agree with @DV7Art  comment. we should not catch and throw an RuntimeException, please repeat the theoretical material.